pages/contact-us/contact-employment-insurance.js

```diff
@@ -0,0 +1,249 @@
+  tmp.fr.mail.details = tmp.fr.mail.details.map((x) => {
+    return {
+      province: x.label,
+      id: x.detail,
+      contentEi:
+        'Service Canada\n\nEmployment Insurance Program\n\nPO Box 2100\n\nVanvouver BC V6B 3T4',
+      contentDocuments:
+        'Service Canada Center\n\nPO Box 245\nEdmonton AB T5J 2J1',
+    }
+  })
+  tmp.en.methods.pop()
+  tmp.fr.methods.pop()
+  return tmp
+}
+
+export default function ContactEmploymentInsurance(props) {
+  /* istanbul ignore next */
+  const t = props.locale === 'en' ? en : fr
+
+  const [openModalWithLink, setOpenModalWithLink] = React.useState({
+    isOpen: false,
+    activeLink: '/',
+  })
+
+  function openModal(link) {
+    setOpenModalWithLink({ isOpen: true, activeLink: link })
+  }
+
+  function closeModal() {
+    setOpenModalWithLink({ isOpen: false, activeLink: '/' })
+  }
+
+  return (
+    <div id="homeContent" data-testid="homeContent-test">
+      <Heading id="my-dashboard-heading" title={props.contactMethods.title} />
+      <div className="py-5" />
+      <TableContent
+        sectionList={[
+          ...props.contactMethods.methods,
+          { title: 'Mail', id: 'mail' },
+        ].map((item, i) => {
+          return { name: item.title, link: `#${item.id}` }
+        })}
+      />
+
+      {props.contactMethods.methods.map((item, i) => (
+        <Fragment key={i}>
+          <ContactSection programUniqueId={i} {...item} />
+        </Fragment>
+      ))}
+      <div className="max-w-3xl" id="mail">
+        <h2 className="py-4 md:py-9 md:mt-2 text-4xl font-display font-bold">
+          {props.contactMethods.mail.title}
+        </h2>
+        <div className="list-disc list-inside list-disc markdown_div pb-4">
+          <Markdown>{props.contactMethods.mail.intro}</Markdown>
+        </div>
+        {props.contactMethods.mail.details
+          .filter((x) => x.province && x.contentEi && x.contentDocuments)
+          .map((item, i) => (
+            <ContactProvince {...item} locale={props.locale} key={i} />
+          ))}
+      </div>
+
+      <Modal
+        className="flex justify-center bg-black/75 h-full"
+        isOpen={openModalWithLink.isOpen}
+        onRequestClose={closeModal}
+        contentLabel={t.aria_exit_beta_modal}
+      >
+        <ExitBetaModal
+          closeModal={closeModal}
+          closeModalAria={t.close_modal}
+          continueLink={openModalWithLink.activeLink}
+          popupId={props.popupContent.popupId}
+          popupTitle={props.popupContent.popupTitle}
+          popupDescription={props.popupContent.popupDescription}
+          popupPrimaryBtn={props.popupContent.popupPrimaryBtn}
+          popupSecondaryBtn={props.popupContent.popupSecondaryBtn}
+        />
+      </Modal>
+    </div>
+  )
+}
+
+export async function getStaticProps({ res, locale }) {
+  const content = await getProfileContent().catch((error) => {
+    logger.error(error)
+    //res.statusCode = 500
+    throw error
+  })
```

Comment:
I don't believe the profile content is being used here

Reply:
removed